stl/inc/yvals_core.h

```diff
@@ -364,6 +364,12 @@
 
 #include <vcruntime.h>
 
+#ifdef _FORBID_ALL_STL_HEADERS
+static_assert(false, "error STL2000: You tried to include a C++ standard header file. "

```

Comment:
Similarly "Tried to include a C++ Standard Library header file with _FORBID_ALL_STL_HEADERS defined". The "did you mean _ENFORCE_ONLY_CORE_HEADERS" seems totaly unnecessary. It's not like these two macros are similar enough to be easily mistaken with each other, they literally do what they say.

Reply:
I added because of this:

https://github.com/microsoft/STL/issues/311

>We could also add a mechanism to completely block the STL (presumably, a macro that <yvals_core.h> checks for). The error message for that could recommend blocking only non-core features.

But I don't know. Maybe it's not needed...